src/pages/generic/supportedFunctionality.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div>
+    <h2>{{ title }}</h2>
+    With these functions you are able to check supported events, functions and requests.
+    <br><br>
+    <div>
+      <h3>Supported events</h3>
+      <p>
+        With <InlineCode>getSupportedEvents</InlineCode> you get a list of events the map supports<br>
+        <RunExampleButton @click="getSupportedEvents">Get supported events</RunExampleButton>
+        <br><br>
+        <CodeSnippet snippet="channel.getSupportedEvents(function(data){
+          channel.log('GetSupportedEvents: ', data);
+        });"></CodeSnippet>
+      </p>
+    </div>
+    <br>
+
+    <div>
+      <h3>Supported functions</h3>
+      <p>
+        With <InlineCode>getSupportedFunctions</InlineCode> you get a list of functions the map supports.<br>
+        <RunExampleButton @click="getSupportedFunctions">Get supported functions</RunExampleButton>
+        <br><br>
+        <CodeSnippet snippet="channel.getSupportedFunctions(function(data){
+          channel.log('GetSupportedFunctions: ', data);
+        });"></CodeSnippet>
+      </p>
+    </div>
+    <br>
+
+    <div>
+      <h3>Supported requests</h3>
+      <p>
+        With <InlineCode>getSupportedRequests</InlineCode> you get a list of requests the map supports.<br>
+        <RunExampleButton @click="getSupportedRequests">Get supported requests</RunExampleButton>
+        <br><br>
+        <CodeSnippet snippet="channel.getSupportedRequests(function(data){
+          channel.log('GetSupportedRequests: ', data);
+        });"></CodeSnippet>
+      </p>
+    </div>
+    <br>
+
+    <div>
+      <h3>Get RPC client version</h3>
+      <p>
+        With <InlineCode>isSupported</InlineCode> you can check that the iframe gives us an expected version of Oskari RPC client.
+        <br>
+        <RunExampleButton @click="checkClientSupport">Check client version support</RunExampleButton>
+
+        <br>
+        <br>
+        <CodeSnippet>
+    channel.onReady(function() {
+      channel.log('Map is now listening'); //channel is now ready and listening. 
+      var expectedOskariVersion = '2.3.1';
```

Comment:
Fill in version from the same constant that is used "for real" so we don't have to maintain the version in two places.